File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Cpp/pattern4.cpp
/*Program 4: Write a Program to Print following Pattern
*
* *
* * * 
* * * * 
*/
#include<iostream>


int main(){

	for(int outer = 1 ; outer <=4; outer++ ){

		for(int inner = 1 ; inner <=outer; inner++){
			
			printf("* ");
		}		
		printf("\n");
	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Java/Pattern4.java
/*Program 4: Write a Program to Print following Pattern
*
* *
* * * 
* * * * 

*/

class Patter4{
	
	public static void main(String args[]){

		for(int outer = 1 ; outer <=4; outer++ ){

			for(int inner = 1 ; inner <=outer; inner++){
			
				System.out.printf("* ");
			}		
			System.out.printf("\n");
		}
	}	

}	
